Source/Module/AnalogClipperModule/Clipper.cpp

```diff
@@ -0,0 +1,103 @@
+//
+//  Clipper.cpp
+//  clipper - VST3
+//
+//  Created by Francesco Magoga
+//
+
+#include "Clipper.h"
+
+#include <cmath>
+
+#include <unistd.h>
+
+#include <juce_dsp/maths/juce_FastMathApproximations.h>
+
+#define THRESHOLD 1e-4				//in Volts
+#define MAX_ITERATIONS 250
+
+using juce_math = juce::dsp::FastMathApproximations;
+
+Clipper::Clipper()
+{
+	diode.alpha = 1/(2*23e-3);         //23mV
+	diode.beta = 2.52e-9;              //2.52nA
+	
+	Rin = 1e3;                          //1kOhm
+	C = 100e-9;                         //100nF
+		
+	h = 1e-5;
```

Comment:
Check if the h variable is a constant or changes according to sampleRate/T parameters